concepts/default-arguments/introduction.md

```diff
@@ -10,4 +10,12 @@ When compiled, Elixir creates a function definition for `number/0` (no arguments
 
 If more than one argument has default values, the default values will be applied to the function from left to right to fill in for missing arguments.
 
+If the function has more than one clause, the default arguments should be defined in a function header (a function without a body) before the function clauses:
+
+```elixir
+def number(n \\ 13)
+def number(n) when n < 10, do: "Dream bigger!"
+def number(n) when n > 100, do: "Not that big..."
+```
+
```

Comment:
This is the case in the exercise (guessing game) and it might be not trivial for people to figure it out on their own, especially if they don't see compilation warnings